Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM opensciencegrid/software-base:3.6-el8-release
+RUN dnf install -y python3.11
+ADD test-resources/etc /etc
+ADD test-resources/var /var
+WORKDIR /script
+COPY ./gracc-backup.py .
+ENTRYPOINT ["python3.11", "./gracc-backup.py", "raw"]
```

Comment:
This should probably be just 
ENTRYPOINT ["python3.11", "./gracc-backup.py"]

Then, the "type" of backup will be passed as command line arguments to the docker run command in the .service file.